resources/generate_book.py

```diff
@@ -298,13 +298,41 @@ def make_book(rm: ResourceManager, i18n: I18n, local_instance: bool = False):
+            entity('tfc:boar', 'The boar spawns in plains with temperatures below 25°C, and rainfalls between 130 and 400mm.', 'Boar'),
+            entity('tfc:rabbit', 'The rabbit is known to chew on carrots and cabbage. They are ubiquitous in the world, changing their coat based on climate. They only need 15mm of rain to spawn.', 'Rabbit'),
+            entity('tfc:fox', 'The fox likes to eat the berries off of bushes. It can be found in forests with temperatures below 25°C, and rainfalls between 130 and 400mm.', 'Fox'),
+            text('$(thing)Shore Animals$() only spawn on sea shores and spend some of their day swimming, and some walking on the beach. They are curious creatures, and will follow the player around, but cannot be tamed.'),
+            entity('tfc:penguin', 'The penguin spawns in only the coldest beaches, with temperatures of at most -14°C and rainfalls of at least 75mm.', 'Penguin'),
+            entity('tfc:turtle', 'The sea turtle likes warm water. It spawns in temperatures of at least 21°C and rainfalls of at least 250mm.', 'Sea Turtle'),
+            text('$(thing)Fish$() are small creatures that swim in water. Most of them can be $(l:mechanics/fishing)fished$(). Some prefer oceans, rivers, or lakes.'),
+            entity('tfc:cod', 'Cod prefer colder oceans, at most 18°C. They can be fished.', 'Cod'),
+            entity('tfc:pufferfish', 'Pufferfish live in any ocean with at least a temperature of 10°C.', 'Pufferfish'),
+            entity('tfc:jellyfish', 'Jellyfish live in warmer oceans, with a temperature of at least 18°C.', 'Jellyfish'),
+            entity('tfc:tropical_fish', 'Tropical fish prefer warmer oceans, with a temperature of at least 18°C.', 'Tropical Fish'),
+            entity('tfc:salmon', 'Salmon spawn in any river or lake with a temperature of at least -5°C.'),
+            entity('tfc:bluegill', 'Bluegill spawn in any river or lake with a temperature of at least -10°C and at most 26°C.'),
+            text('$(thing)Shellfish$() are small animals that live on the floor of bodies of water. They cannot be fished, but drop shells that can be eaten or made into $(l:mechanics/flux)flux$().').anchor('shellfish'),
+            entity('tfc:isopod', 'Isopods spawn in deeper sections of oceans of at most 14°C.', 'Isopod'),
+            entity('tfc:lobster', 'Lobster spawn in any ocean that is at most of a temperature of 21°C.', 'Lobster'),
+            entity('tfc:crayfish', 'Crayfish are like lobster, but spawn in rivers and lakes. They need a temperature of at least 5°C and a rainfall of at least 125mm.', 'Crayfish'),
+            entity('tfc:horseshoe_crab', 'Horseshoe crabs spawn in oceans of moderate climate, between 10 and 21°C and with a rainfall of at most 400mm.', 'Horseshoe Crab'),
+            text('$(thing)Water Creatures$() are larger animals that live in the bigger bodies of water. Some of them predate on fish. They drop $(l:mechanics/lamps)blubber$(), which can be made into lamp fuel.'),
```

Comment:
The link here looks like it should link to an anchor i.e. `mechanics/lamps#blubber`, or it should have `$(thing)blubber$(), which can be made into a$(l:...)lamp fuel$()`